package.json

```diff
@@ -3,10 +3,10 @@
   "version": "1.0.6",
   "preferGlobal": true,
   "description": "A Bash script to build a SVG sprite from a folder of SVG files (typically icons).",
-  "bin": "bin/spritesh",
+  "bin": "bin/npm/spritesh",
```

Comment:
I see. Well we can leave the bin/spritesh as the main script and then copy it to bin/gem/sprite.sh for use with the ruby wrapper.